scripts/depcheck/missing.ts

```diff
@@ -45,7 +45,13 @@ const IGNORE_MATCHES = {
     'tailwindcss',
   ],
   cli: ['nx'],
-  cypress: ['cypress', '@angular-devkit/schematics', '@nrwl/cypress'],
+  cypress: [
+    'cypress',
+    '@angular-devkit/schematics',
+    '@nrwl/cypress',
```

Comment:
This... should not be necessary?

Reply:
required from plugins which is run-time vs buildtime so it can be safely ignored.